02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -72,10 +81,12 @@ As a result, others who read the resulting documentation will consider it "poor"
 
 ### Maintaining Documentation's Accuracy
 
-Third, documentation becomes outdated/inaccurate due to changes, especially breaking changes. It does this in two ways: decreasing its usefulness (depends on the 'size' of a change) and decreasing its coherence (depends on the 'frequency' of changes).
+Third, documentation becomes outdated/inaccurate due to changes in the subject of the documentation, especially breaking changes. Two aspects of changes: the 'size' of a change in the subject, and the 'frequency' of these changes.
```

Comment:
Yeah, I agree with that.

What do you think about this?

Rather than

"...due to changes"

it says

"...due to changes in the subject of the documentation..."

I'd prefer the latter because, as I understand it, the problem is the documentation becoming out-of-sync with the thing being documented, but the former way of saying it makes me think the problem is changes in the documentation itself. Am I misunderstanding that?

Reply:
> I'd prefer the latter because, as I understand it, the problem is the documentation becoming out-of-sync with the thing being documented, but the former way of saying it makes me think the problem is changes in the documentation itself. Am I misunderstanding that?

Ah... I see your point now. That's a good point.
I think what trips me up then is seeing 'documentation' appear twice in the same sentence. Perhaps it could be changed to 'documentation becomes inaccurate/outdated when the thing/subject being documented changes' 
Or 'documentation becomes inaccurate/outdated when changes occur in the thing being documented.'  ?

Reply:
Yeah, I think that would be more clear, for sure. I have a habit of using "proper" grammar, often going so far as to make smooth reading more difficult, regrettably. I'll change that now.

Reply:
Yeah.... it's frustrating isn't it? Why can't people just read properly?